core/src/main/java/org/infinispan/marshall/core/ExternallyMarshallable.java

```diff
@@ -2,18 +2,18 @@
 
 import java.io.Serializable;
 
-// Temporary class, should become active by default when testing
 final class ExternallyMarshallable {
 
    private ExternallyMarshallable() {
       // Static class
    }
 
-   public static boolean isAllowed(Object obj) {
+   static boolean isAllowed(Object obj) {
       String pkg = obj.getClass().getPackage().getName();
       boolean isBlackList =
             obj instanceof Serializable
             && isMarshallablePackage(pkg)
+            && !(obj instanceof ExternalPojo)
             && !isWhiteList(obj.getClass().getName());
```

Comment:
> hmmm... what is the problem that ExternalPojo tries to solve?

Detect Infinispan classes for which we should have externalizers. If we forget that a class has a externalizer, and it uses standard Serialization, this check will catch it.

If we really expect it to be Serializable, e.g. testing Serializable classes, then the class can be made to extend ExternalPojo interface as marker.